index.html

```diff
@@ -135,10 +163,13 @@ <h1 class="item1 h1">I’m Kirubel. Glad to see you!</h1>
             <li class="item1"><a href="#"><img src="./images/angellist.png" alt=" adgelist logo image"></a></li>
           </div>
           <button>Get My Resume</button>
+        </div>
+        <div class="about12 about124">
           <div class="abfr">
             <p class="language">Language</p>
             <img class="abfr1" src="./images/download.png" alt="droupdown icon">
           </div>
+          <div class="auto">
           <div class="abla abla1">
             <img class="abimg" src="./images/Ellipse 1.png" alt="JS image">
             <p>JavaScript</p>
```

Comment:
:orange_circle: [OPTIONAL] Please make sure the bottom borders are applied to each item to match the Figma Design. 
Your Design
![missing border](https://user-images.githubusercontent.com/26736582/229287926-38c47142-6b10-4e5a-a500-9e1fb62994db.png)
Figma Design
![figmaborder](https://user-images.githubusercontent.com/26736582/229287938-14aa52e4-6377-40ef-973b-ba67e139d242.png)

Also, Note that for semantics reasons you should use
```
 <ul>
<li>Languages.
     <ul>
         <li>Javascript</li>
        <li>CSS</li>
       <li>HTML</li>
   </ul>
</li>
<li>Frameworks</li>
<li>Skills</li>
</ul>
```